ProjectName/.husky/pre-commit

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx tsc
-npx lint-staged
+yarn tsc
+# yarn prettier:fix
```

Comment:
What I suggest is having `lint` and `typescript` check for `pre-commit` and `tests` running for pre-push (but since we did not introduce tests, this part could be skipped).

Reply:
I have put the lint-staged in husky pre-commit hook (in package.json file) ;] But when I try to add yarn tsc in the lint-staged command there would be an error so I leave it there